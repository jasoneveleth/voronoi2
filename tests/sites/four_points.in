0.3244, 0.4234
0.85645, 0.325
0.2344, 0.8766
0.8657, 0.9987
