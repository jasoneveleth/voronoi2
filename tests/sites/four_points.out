ec0b02fcb8e52823bd493e1f1f86bb9f  -
